### PR TITLE
fixed names of links to joining the orgs

### PR DESCRIPTION
**Pull request by indiesaudi**

### CODE REVIEW

**Comment by mairaw** [comment]
@terrajobst are those links correct? It seems a bit weird that the link to join the Microsoft team would have dotnet in it. But @indiesaudi changes make sense to me

**Comment by omajid** [comment]
Check this history on this file. It seems to be a great source of confusion:

https://github.com/dotnet/core/pull/3957
https://github.com/dotnet/core/pull/4363
https://github.com/dotnet/core/pull/3401

cc @richlander

File: Documentation/microsoft-team.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ You need to link your GitHub and @microsoft.com accounts. Click the link:
 
 You need to join teams in two organizations. Click the two links:
 
-* [dotnet org](https://repos.opensource.microsoft.com/dotnet/teams/microsoft/join/)
-* [Microsoft org](https://repos.opensource.microsoft.com/microsoft/teams/dotnet/join/)
+* [Microsoft org](https://repos.opensource.microsoft.com/dotnet/teams/microsoft/join/)
```
Comment (mairaw) [inline Documentation/microsoft-team.md:13]:
```suggestion
* [Microsoft org](https://repos.opensource.microsoft.com/microsoft/join/)
```

File: Documentation/microsoft-team.md [hunk 2]
```diff
@@ -10,8 +10,8 @@ You need to link your GitHub and @microsoft.com accounts. Click the link:
...
+* [dotnet org](https://repos.opensource.microsoft.com/microsoft/teams/dotnet/join/)
```
Comment (mairaw) [inline Documentation/microsoft-team.md:14]:
```suggestion
* [dotnet org](https://repos.opensource.microsoft.com/dotnet/join/)
```

**Review by mairaw — COMMENTED** [review]
I think this would be the right links. at least I shared the ones for dotnet several times and it seems to work

**Comment by terrajobst** [comment]
I pushed cc7ec86 to (hopefully) address this

**Comment by mairaw** [comment]
I think it's still confusing. Hard to repro without being a new employee, but when I go to https://repos.opensource.microsoft.com/Microsoft/teams/dotnet/join/, I see 
image

Whenever someone asks me how they can see the website repo, I send them this link instead: https://repos.opensource.microsoft.com/dotnet/join/

**Comment by Lior-Koren** [comment]
thanks

**Pull request closed without merging**